Dotnet.Homeworks.DataAccess/Repositories/OrderRepository.cs

```diff
@@ -0,0 +1,32 @@
+using Dotnet.Homeworks.Domain.Abstractions.Repositories;
+using Dotnet.Homeworks.Domain.Entities;
+
+namespace Dotnet.Homeworks.DataAccess.Repositories;
+
+public class OrderRepository : IOrderRepository
+{
+    public async Task<IEnumerable<Order>> GetAllOrdersFromUserAsync(Guid userId, CancellationToken cancellationToken)
```

Comment:
```suggestion
    public async Task<IEnumerable<Order>> GetAllOrdersFromUserAsync(Guid userId, CancellationToken cancellationToken = default)
```
Может быть так со всеми сделать?